include/clang/Frontend/CompilerInstance.h

```diff
@@ -412,6 +417,14 @@ class CompilerInstance : public ModuleLoader {
     VirtualFileSystem = std::move(FS);
   }
 
+  IntrusiveRefCntPtr<clang::InMemoryOutputFileSystem> getInMemoryOutputFileSystem() const {
```

Comment:
Judging by the comments above and below, perhaps it would be appropriate to say:

```
  /// }
  /// @name In-Memory Output File System
  /// {
```

Reply:
done